.github/workflows/pre-release.yml

```diff
@@ -0,0 +1,45 @@
+name: Release
+
+on:
+  push:
+    branches:
+      - main
+
+jobs:
+  logging:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Print event metadata
+        run: |
+          echo "${{toJSON(github.event.commits[0])}}"
+          echo "${{toJSON(github.event.head_commit)}}"
+  pre-release:
+    if: |
+      !contains(github.event.head_commit.message, 'scalameta/dependabot/') &&
+      !contains(github.event.head_commit.message, 'Signed-off-by: dependabot[bot]') &&
+      github.event.author.email != 'bot@scalameta.org'
```

Comment:
```shell
!contains(github.event.head_commit.message, 'scalameta/dependabot/') &&
 !contains(github.event.head_commit.message, 'Signed-off-by: dependabot[bot]') &&
 ```
is not nice but it should be good enough for start. I'm not sure that dependabot has some constant email that ca be used. 
`logging` workflow should show what properties can be used to filter out dependabot PR's more gracefully. 

Alternatively, I can leave only `!contains(github.event.head_commit.message, 'scalameta/dependabot/')` which should be sufficient and don't bother with possible better alternatives.

@tgodzik it's your call